charts/seq/templates/deployment.yaml

```diff
@@ -46,6 +46,9 @@ spec:
 {{- if .Values.firstRunRequireAuthenticationForHttpIngestion }}
             - name: "SEQ_FIRSTRUN_REQUIREAUTHENTICATIONFORHTTPINGESTION"
               value: "{{ .Values.firstRunRequireAuthenticationForHttpIngestion }}"
+{{- end }}
+{{- if .Values.addiotnalEnv }}
```

Comment:
seems you had a spelling error of additionalEnv here?

Reply:
Oh well spotted! I'll get this patched ASAP